cpp/modmesh/view/RManager.cpp

```diff
@@ -281,6 +202,182 @@ void RManager::setUpMenu()
             []() {}));
 }
 
+void RManager::setUpCameraControllersMenuItems() const
+{
+    auto * use_orbit_camera = new RAction(
+        QString("Use Orbit Camera Controller"),
+        QString("Use Oribt Camera Controller"),
+        [this]()
+        {
+            qDebug() << "Use Orbit Camera Controller (menu demo)";
+            for (auto subwin : m_mdiArea->subWindowList())
+            {
+                auto * viewer = dynamic_cast<R3DWidget *>(subwin->widget());
+
+                if (viewer == nullptr)
+                    continue;
+
+                viewer->scene()->setOrbitCameraController();
+                viewer->scene()->controller()->setCamera(viewer->camera());
+            }
+        });
+
+    auto * use_fps_camera = new RAction(
+        QString("Use First Person Camera Controller"),
+        QString("Use First Person Camera Controller"),
+        [this]()
+        {
+            qDebug() << "Use First Person Camera Controller (menu demo)";
+            for (auto subwin : m_mdiArea->subWindowList())
+            {
+                auto * viewer = dynamic_cast<R3DWidget *>(subwin->widget());
+
+                if (viewer == nullptr)
+                    continue;
+
+                viewer->scene()->setFirstPersonCameraController();
+                viewer->scene()->controller()->setCamera(viewer->camera());
+            }
+        });
+
+    auto * cameraGroup = new QActionGroup(m_mainWindow);
+    cameraGroup->addAction(use_orbit_camera);
+    cameraGroup->addAction(use_fps_camera);
+
+    use_orbit_camera->setCheckable(true);
+    use_fps_camera->setCheckable(true);
+    use_orbit_camera->setChecked(true);
+
+    m_viewMenu->addAction(use_orbit_camera);
+    m_viewMenu->addAction(use_fps_camera);
+}
+
+void RManager::setUpCameraMovementMenuItems() const
+{
+    auto * reset_camera = new RAction(
+        QString("Reset (esc)"),
+        QString("Reset (esc)"),
+        [this]()
+        {
+            const auto * subwin = m_mdiArea->currentSubWindow();
+            if (subwin == nullptr)
+                return;
+
+            auto * viewer = dynamic_cast<R3DWidget *>(subwin->widget());
+            if (viewer == nullptr || viewer->camera() == nullptr)
+                return;
+
+            viewer->resetCamera();
+        });
+
+    auto * move_camera_up = new RAction(
+        QString("Move camera up (W/⬆)"),
+        QString("Move camera up (W/⬆)"),
+        createCameraMovementItemHandler([](CameraInputState & input)
+                                        { input.tyAxisValue = 1.0; }));
```

Comment:
View submenu items for camera movement added here

Reply:
I wonder if we should keep all the items in a "folder" (not sure what is the proper name of it) in the menu?

Reply:
How about "Camera movement" ?

Reply:
> How about "Camera movement" ?

Sounds OK, but I'd like to make it shorter.  Could you please call it "Camera move"?